File: API/Models/StationFare.cs
namespace API.Models;

public class StationFare
{
    public string Id { get; set; }
    public string Price { get; set; }
    public string From { get; set; }
    public string Destination { get; set; }

    public IList<Ticketing> Ticketing { get; set; } = new List<Ticketing>();
}

File: client/src/app/api/models/station-fare.ts
/* tslint:disable */
/* eslint-disable */
import { Ticketing } from '../models/ticketing';
export interface StationFare {
  destination?: string | null;
  from?: string | null;
  id?: string | null;
  price?: string | null;
  ticketing?: Array<Ticketing> | null;
}
